### DISCUSSION TITLE
Change the latex env that represents markdown blockquote

### DISCUSSION THREAD

**Discussion by avidseeker** [jgm/pandoc#9847] [Q&A]
pandoc translates markdown blockquotes `>` into `\begin{quote}...\end{quote}`.

How do I change that to `\begin{quotation}...\end{quotation}`?

**Comment by jgm** [comment]
Easiest approach would be to redefine the `quote` environment in your preamble.  E.g. in markdown metadata,
````
header-includes: |
  ```{=latex}
  \renewenvironment{quote}{\begin{quotation}}{\end{quotation}}
  ```
````

Slightly harder would be to write a Lua filter that matches BlockQuote elements and outputs their contents surrounded by the appropriate raw LaTeX begin and end commands.
